fastlane/lib/fastlane/actions/create_pull_request.rb

```diff
@@ -33,6 +33,27 @@ def self.run(params)
           html_url = json['html_url']
           UI.success("Successfully created pull request ##{number}. You can see it at '#{html_url}'")
 
+          if params[:labels]
+            payload = {
+              "labels": params[:labels]
+            }
+            GithubApiAction.run(
+              server_url: params[:api_url],
+              api_token: params[:api_token],
+              http_method: 'PATCH',
+              path: "repos/#{params[:repo]}/issues/#{number}",
```

Comment:
Would it be `pulls` rather than `issues`?

Reply:
Nope nope! A pull request is technically also an issue 🙃